src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -205,7 +205,7 @@ def choose_group_quarters(self, target_number_sims: int) -> pd.Series:
             options=group_quarters,
             n_to_choose=target_number_sims,
             randomness_stream=self.randomness,
-            weights=self.population_data["households"][["person_weight"]],
+            weights=self.population_data["households"]["person_weight"],
```

Comment:
Very curious how this wasn't breaking before since this would be a dataframe.

Reply:
To be honest, I am not sure. Something about using `vectorize` there.

Reply:
Or possibly `np.take` can transparently use a single-column dataframe, treating it like a 1d array.

Reply:
Ah, it was the first one: each call to the lambda inside `vectorize` returns a single-element pandas Series, but the vectorize function flattens each result into a simple integer data type (using the one value in the Series, and discarding the index) to return a numpy array.